File: C/Biblioteca.c
#include <stdbool.h>
typedef struct {
    char titulo[100];
    char autor[100];
    int anoPublicacao;
    int numCopiasDisponiveis;
} Livro;

void emprestarLivro(Livro *livro);
void devolverLivro(Livro *livro);
bool verificarDisponibilidade(Livro *livro);
void obterInformacoesLivro(Livro *livro);

File: haskell/Biblioteca.hs
data Livro = Livro {
    titulo :: String,
    autor :: String,
    anoPublicacao :: Int,
    numCopiasDisponiveis :: Int
}

emprestarLivro :: Livro -> Livro
emprestarLivro livro = livro

devolverLivro :: Livro -> Livro
devolverLivro livro = livro

verificarDisponibilidade :: Livro -> Bool
verificarDisponibilidade livro = False

obterInformacoesLivro :: Livro -> String
obterInformacoesLivro livro = ""
